angular-tsconfig.json

```diff
@@ -5,7 +5,8 @@
   "angularCompilerOptions": {
     // In snapshot builds the compiler-cli will now use ngtsc by default. In
     // order to be able to build the summary files, we need to use ngc.
-    "enableIvy": false
+    "enableIvy": false,
+    "disableTypeScriptVersionCheck": true
```

Comment:
These are temporarily needed because the Angular compiler does not support TS 3.9 yet.

Note: We need to set them multiple times because `@angular/bazel` does not inherit this option.

Reply:
This is resolved.